packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -313,6 +323,49 @@ describe('FLE tests', () => {
       expect(collections).to.not.include('enxcol_.collfle2.ecoc');
       expect(collections).to.not.include('collfle2');
     });
+
+    it('allows compactStructuredEncryptionData command when mongo instance configured with auto encryption', async() => {
+      const shell = TestShell.start({
+        args: ['--nodb'],
+        env: {
+          ...process.env,
+          MONGOSH_FLE2_SUPPORT: 'true'
+        },
+      });
```

Comment:
to account for the updates to the main branch:

```suggestion
      const shell = TestShell.start({
        args: ['--nodb', `--csfleLibraryPath=${csfleLibrary}`]
      });
```